addon/components/denali-progress.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2021, Verizon Media
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Component from '@glimmer/component';
+import { arg } from 'ember-arg-types';
+import { boolean, oneOf } from 'prop-types';
+import { SIZES } from './denali-progress-enums';
+
+export default class DenaliProgressComponent extends Component {
+  @arg(oneOf(SIZES))
+  size = SIZES[0];
+
+  @arg(boolean)
+  isLoading = false;
+
```

Comment:
Can we add `color` and `shade` as args?

Reply:
As per previous comments, we removed value, color and shade args from denali-progress.js since they are already part of denali-progress/bar.js

Reply:
@jkusa  ??